src/algebra/ordered_monoid_lemmas.lean

```diff
@@ -708,17 +710,31 @@ lemma strict_mono.mul_const' (hf : strict_mono f) (c : α) :
 
 end right
 
+/--  The product of two strictly monotone functions is strictly monotone. -/
+@[to_additive
+"The sum of two strictly monotone functions is strictly monotone."]
+lemma strict_mono.mul_strict_mono [has_lt β] [preorder α]
```

Comment:
I'd prefer the name `strict_mono.mul'`, matching the naming scheme for `strict_mono.mul`.

Reply:
Floris, thanks!  I changed the name, and used the unprimed version for the additive one!